sdcm/utils/gce_utils.py

```diff
@@ -30,11 +30,14 @@
 
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
+    assert region.startswith("us-east1") or region.startswith("us-west1"), "only supports two regions"
+    if region.count("-") != 1:
+        return region
+    if region.startswith("us-east1"):
```

Comment:
```suggestion
    if region == "us-east1":
```

Reply:
Maybe because in GCE the region include the zone?
Like "us-east1b"

Reply:
> Maybe because in GCE the region include the zone?
> Like "us-east1-b" ?

Yes, we didn't prevent to include available zone suffix in gce_datacenter.

```
defaults/gce_config.yaml:gce_datacenter: 'us-east1'
defaults/k8s_gce_minikube_config.yaml:gce_datacenter: 'us-east1'
defaults/k8s_gke_config.yaml:gce_datacenter: 'us-east1-b'
docs/configuration_options.md:| **<a href="#user-content-gce_datacenter" name="gce_datacenter">gce_datacenter</a>**  | Supported: us-east1 - means that the zone will be selected automatically or you can mention the zone explicitly, for example: us-east1-b | N/A | SCT_GCE_DATACENTER
internal_test_data/complex_test_case_with_version.yaml:gce_datacenter: 'us-east1-b'
```